docs/docs/features/conditions.md

```
@@ -11,7 +7,7 @@ The **Conditions** page provides information about the configured conditions in
 
 ---
 
-## TAB: Negative Matches{#negative}
+## TAB: Negative Matches
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)